docs/howtoguides/how_to_know_if_system_affected_by_a_list_of_cves.rst

```diff
@@ -0,0 +1,52 @@
+.. _pro-check-list-cves:
+
+Given a list of CVEs, how to know if you are affected by them?
+**************************************************************
+
+If you have a list of Common Vulnerabilities and Exposures (CVE) or Ubuntu Security Notifications (USN) and want to check if your Ubuntu system is affected by them, you can easily check using the pro fix API. In this guide we'll show you how to leverage the pro fix API:
+
+.. note::
+    The `pro fix` API is provided as a part of the `pro` package, which is a security tool for Ubuntu systems. If you don't have it installed, you can install it by following the instructions in the `pro` package documentation.
+
+Using the `pro.fix.cve.plan` API
+--------------------------------
+
+To check if your system is affected by a list of CVEs, you would need to use the ``u.pro.security.fix.cve.plan.v1`` endpoint as the
+CLI `pro fix --dry-run` command does not support multiple CVEs. However, as can be seen :ref:`in the endpoint documentation <cve-execute-api-v1>`,
+this API endpoint will output a JSON blob containing the current status of each CVE.
+
+This JSON output does not help with easy visualisation of the current status of each CVE. To address this issue, we will use a ``jq`` filter.
+The ``jq`` command can easily parse JSON data directly in the terminal, allowing for a more readable output. To know more about it, please refer to `the jq manpage <https://manpages.ubuntu.com/manpages/xenial/man1/jq.1.html>`_
+
+First, we would need to install ``jq`` by running the following command:
+
+.. code-block:: bash
+
+    $ apt updated & apt install jq -y
+
+Once ``jq`` is installed, we can easily parse the JSON data returned from the plan API.
+
+For example, if we want to see if our system is affected by the following CVEs: **CVE-2020-28196, CVE-2020-15180**
+and **CVE-2017-9233**.
+
+We make use of the plan API by running the following command:
+
+.. code-block:: bash
+
+    pro api u.pro.security.fix.cve.plan.v1 --data '{"cves": ["CVE-2020-28196", "CVE-2020-15180", "CVE-2017-9233"]}' \ 
+        | jq -r '.data.attributes.cves_data.cves[] | "\(.title) (\(.description)) - Current Status: \(.current_status)"'
+
+This command will return an output that will have the following structure:
+
+.. code-block:: bash
+
+    CVE-2020-28196 (Kerberos vulnerability) - Current Status: not-affected
+    CVE-2020-15180 (MariaDB vulnerabilities) - Current Status: not-affected
+    CVE-2017-9233 (Coin3D vulnerability) - Current Status: not-affected
+
+Note that each entry in this output consists of three fields:
+
+* **CVE NAME**: The name of the CVE
+* **CVE DESCRIPTION**: The description of the CVE
+* **CVE STATUS**: The current status of the CVE
+
```

Comment:
We might want to link here to the CVE database page where users can look up the CVEs/USNs to get more info than what's presented by the Pro Client